README.md

```diff
@@ -13,7 +13,7 @@ framework.
 Running tests requires two environment variables to be are set:
 ```
 export PITCHFORK_TOOLNAME=trident
-export PITCHFORK_CONFROOT=/Users/${USER}/git/trident/tconf/
+export PITCHFORK_CONFROOT=/${HOME}/repositories/path/trident/tconf/
```

Comment:
Better strip the leading / otherwise this becomes //home/jeroen//repositories in many cases. Then again, that does not hurt ;)